docs/i_am/data_scientist.rst

```diff
@@ -23,7 +24,8 @@ Example: English Word Checker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Let's work on a really trivial example to help you understand how to use fdep.
-How about building a word list from an English book and determining if the given word is an English word?
+How about building a word list from a book and determining if the
```

Comment:
The idea was to do a somewhat incomplete/trivial example of extracting all the used English words from a book written in English and be able to tell if a given word is an English word or not. Maybe we need a better example for this section, but I'm okay with this for now.